Python/codecs.c

```diff
@@ -49,36 +49,36 @@ int PyCodec_Register(PyObject *search_function)
     return -1;
 }
 
+extern int _Py_normalize_encoding(const char *, char *, size_t);
+
 /* Convert a string to a normalized Python string: all characters are
-   converted to lower case, spaces are replaced with underscores. */
+   converted to lower case, spaces and hyphens are replaced with underscores. */
```

Comment:
Can you please specify that the string is decoded from UTF-8?

Reply:
done